go/tools/bazel_testing/bazel_testing.go

```diff
@@ -219,6 +219,28 @@ func BazelOutput(args ...string) ([]byte, error) {
 	return stdout.Bytes(), err
 }
 
+// BazelOutputWithInput invokes a bazel command with a list of arguments and
+// an input stream and returns the content of stdout.
+//
+// If the command starts but exits with a non-zero status, a *StderrExitError
+// will be returned which wraps the original *exec.ExitError.
+func BazelOutputWithInput(stdin io.Reader, args ...string) ([]byte, error) {
```

Comment:
Can we implement `BazelOutput` in terms of this new method?

Reply:
Do you want me to just modify BazelOutput so it takes stdin by default and then nil it out in all the calls to it?

Reply:
Ah, no, I just wanted to suggest implementing `BazelOutput` as `BazelOutputWithInput(nil, ...)` so that the two implementations don't deviate over time.